apps/vr-tests-react-components/screener.config.js

```diff
@@ -17,26 +17,31 @@ function getCurrentHash() {
 
   return '';
 }
+/**
+ *
+ * @param {string} screenerApiKey
+ * @param {string} sourceBranchName
+ * @param {string} deployUrl
+ * @param {string} targetBranch
+ * @returns
+ */
+function getConfig(screenerApiKey, sourceBranchName, deployUrl, targetBranch) {
+  const baseBranch = targetBranch ? targetBranch.replace(/^refs\/heads\//, '') : 'master';
+  // https://github.com/screener-io/screener-storybook#additional-configuration-options
+  const config = {
+    projectRepo: 'microsoft/fluentui/react-components',
+    storybookStaticBuildDir: 'dist/storybook',
+    storybookConfigDir: '.storybook',
+    apiKey: screenerApiKey,
+    resolution: '1024x768',
+    baseBranch,
+    failureExitCode: 0,
+    alwaysAcceptBaseBranch: true,
+    ...(sourceBranchName !== 'master' ? { commit: getCurrentHash() } : null),
```

Comment:
there is duplication in every config defined, can we make it DRY ?

Reply:
the configs for v8 and v9 are indeed almost identical, will check if it's possible to use only one file

Reply:
they should live within scripts/screener domain, later on this will be separate package

Reply:
will make this change in the same follow-up PR as the one mentioned here: https://github.com/microsoft/fluentui/pull/24739/files#r969406141